### PR TITLE
feat: ability to increase typescript isolate's max memory

### PR DESCRIPTION
**Pull request by dsherret**
Adds the ability to increase (or decrease) the TypeScript isolate's max memory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,6 +186,8 @@ extension has the following configuration options:
   internal diagnostic information.
 - `deno.lint`: Controls if linting information will be provided by the Deno
   Language Server. _boolean, default `true`_
+- `deno.maxTsServerMemory`: Maximum amount of memory the TypeScript isolate can
```
Comment (dsherret) [inline README.md:189]: The vscode setting name for typescript is `typescript.tsserver.maxTsServerMemory`

File: client/src/commands.ts [hunk 2]
```diff
@@ -211,23 +211,24 @@ export function startLanguageServer(
   };
 
   function getV8Flags() {
-    const envFlags = process.env.DENO_V8_FLAGS ?? "";
-    const hasMemoryFlagInEnv = envFlags.includes("--max-old-space-size") ??
-      false;
+    let v8Flags = process.env.DENO_V8_FLAGS ?? "";
     if (
-      hasMemoryFlagInEnv &&
+      v8Flags.includes("--max-old-space-size") &&
```
Comment (bartlomieju) [inline client/src/commands.ts]: Check for `--max_old_space_size` too

Comment (dsherret) [reply client/src/commands.ts]: Added. They should have done one or the other.

**Review by bartlomieju — APPROVED** [review]
Thanks!

**Pull request merged**